src/rpc/protocol.cpp

```diff
@@ -80,7 +79,7 @@ bool GenerateAuthCookie(std::string *cookie_out)
     const size_t COOKIE_SIZE = 32;
     unsigned char rand_pwd[COOKIE_SIZE];
     GetRandBytes(rand_pwd, COOKIE_SIZE);
-    std::string cookie = COOKIEAUTH_USER + ":" + HexStr(rand_pwd, rand_pwd+COOKIE_SIZE);
+    std::string cookie = COOKIEAUTH_USER + ":" + HexStr(Span<unsigned char>(rand_pwd, COOKIE_SIZE));
```

Comment:
Same, `rand_pwd` is C-style array
```suggestion
    std::string cookie = COOKIEAUTH_USER + ":" + HexStr(rand_pwd);
```